05/index.html

```diff
@@ -0,0 +1,157 @@
+          <br>
+          <label for="Piedras">Acercar unas piedras</label>
+          <input type="checkbox" name="pierdas" id="Piedras">
+          <br>
+          <label for="Palo">Usar un palo</label>
+          <input type="checkbox" name="palo" id="Palo">
+          <br>
+          <br>
+          <label for="color">¡Seleccióna el color de las uvas!</label>
+          <input type="color" id="color">
+          <br>
+          <br>
+          <label for="Arboles">¿Cuantos arboles eran?</label>
+          <input type="number" min="1" max="5" value="0" id="Arboles">
+          <br>
+          <br>
+          <label for="Uvas">¡Escribe otro objeto para poder bajar las uvas!</label>
+          <br>
+          <input type="text" placeholder="¡Escribe algo!" id="Uvas">
+          <br>
+          <br>
+          <label for="Dibujo">¡Dibuja unas uvas y compartelas con nosotros!</label>
+          <br>
+          <input type="file" id="Dibujo">
+          <br>
+          <br>
+          <label for="Suerte">¡Escribe tu numero secreto de la suerte!</label>
+          <br>
+          <input type="password" placeholder="¡Escribe algo!" id="Suerte">
+          <br>
+          <br>
+          <label for="Historia">¿Que tanto te gusto la historia?</label>
+          <input type="range" value="0" min="1" max="10" id="Historia">
+          <br>
+          <br>
+          <label>¡Escribe tu opinión sobre esta historia!</label>
+          <br>
+          <textarea>Nos importa tu opinión</textarea>
+          <br>
+          <input type="image" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJk4Huc-Yherg2Clwmyb_1Y_0136PBJQzgfw&usqp=CAU" width="57" height="57">
+          <button type="button">Botón</button>
+          <hr>
+          <h2>¡Queremos saber mas sobre ti!</h2>
+          <label for="Momento">¿En que momento completas esta encuesta?</label>
+          <br>
+          <br>
+          <input type="datetime-local" id="Momento">
+          <br>
+          <input type="date">
+          <br>
+          <input type="month">
+          <br>
+          <input type="week">
+          <br>
+          <input type="time">
+          <hr>
+          <h2>¿Donde podemos cotactarte?</h2>
+          <label for="Facebook">Facebook link</label>
+          <input type="url" id="Facebook">
+          <br>
+          <label for="Instagran">Instagran link</label>
+          <input type="url" id="Instagran">
+          <br>
+          <label for="Twitter">Twitter link</label>
+          <input type="url" id="Twitter">
+          <br>
+          <label for="E-mail">E-mail</label>
+          <input type="email" id="E-mail">
+          <br>
+          <label for="Teléfono">Teléfono celular</label>
+          <input type="tel" id="Teléfono">
+          <br>
+          <br>
+          <input type="reset">
+          <input type="submit">
+          <h3>¡Gracias por completar nuestra encuesta!</h3>
+        </fieldset>
```

Comment:
Fieldsets are to separate sections inside forms / Los Fieldsets son para separar secciones dentro de formularios